content/blog/building-images-bake/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Creating a custom container image for CloudNativePG v2.0"
+date: 2025-06-17
+draft: false
+image:
+    url: 
+    attribution: 
+author: jgonzalez
+tags:
+ - blog
+ - information
+ - programming
+ - applications
+ - containers
+ - postgresql
+ - postgres
+ - images
+ - tutorial
+ - bake
+ - docker
+summary: Creating a container image for CloudNativePG Operator v2.0
+---
+
+## Summary
+In an almost [two years old blog post]({{% ref "/blog/creating-container-images/" %}}), we explained how
+to build a custom container image for CloudNativePG. After two years, many things have changed in the world of containers.
```

Comment:
OK, this is just very clunky wording. Adding a suggestion.
```suggestion
Almost two years ago, we wrote a [blog post on
building custom container images for CloudNativePG]({{% ref "/blog/creating-container-images/" %}}).
Since then, many things have changed in the world of containers.
```